Directory.Build.props

```diff
@@ -356,19 +353,15 @@
     -->
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
 
-    <!-- Suppress preview message as we are usually using preview SDK versions. -->
-    <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
-
+    <!--
+      OSPlatformConfig and TargetOutputRelPath are used by the test archiving targets in
+      corefx and in our publish test targets (https://github.com/dotnet/buildtools/blob/17f28536524248603108c56a16b5b16dcb5a87ff/src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets#L54)
+      Once we refactor our test targets we should try to break this tight coupling so it is only
+      on the corefx side as these are specific to corefx.
```

Comment:
Just want to let you know that I already changed that and agreed with @safern to introduce the following properties in corefx:

`` 
TestArchiveRoot => artifacts/helix/buildconfiguration (defined in Directory.Build.props)
TestArchiveTestsDir => $(TestArchiveRoot)tests/
TestArchiveRuntimeDir => $(TestArchiveRoot)runtime/`
```